docs/data-structures/infix-to-postfix.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: Infix to Postfix Conversion
+slug: /data-structures/infix-to-postfix
+---
+
+## Introduction
+
+An infix expression is of the form: `operand1 operator operand2`, whereas a postfix expression is of the form `operand1 operand2 operator`. Converting an infix expression to postfix expression is one of the many applications of stacks due to its 'Last in First Out' policy.
```

Comment:
Why does `operand1 operator operand2` need unnecessary highlighting? Same for `operand1 operand2 operator`

Reply:
Changed